.github/workflows/tests.yml

```diff
@@ -0,0 +1,40 @@
+
+name: tests
+
+on:
```

Comment:
I am not an expert with GitHub actions but the current setup will run tests on every branch? I think it should run only on `master`:

```yml
on:
  push:
    branches: [master]
  pull_request:
    branches: [master]
```

Reply:
Updated ✅